equinix/fabric_network_read_schema.go

```diff
@@ -0,0 +1,105 @@
+package equinix
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func readChangeSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"href": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "href",
+		},
+		"uuid": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "UUID of Network",
+		},
+		"type": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "network change type",
+		},
+	}
+}
+
+func readNetworkResourceSchema() map[string]*schema.Schema {
```

Comment:
This schema is more correct for what the create should have been. Though the behaviors on create will be different. Double check that it matches and I'll review again.